include/kwk/detail/abi.hpp

```diff
@@ -12,35 +12,48 @@
 
 // Force a function to be inline
 #if defined(KWK_NO_FORCEINLINE)
-#  define KWK_FORCEINLINE inline
+#  define KWK_TRIVIAL            inline
+#  define KWK_FORCEINLINE        inline
+#  define KWK_LAMBDA_FORCEINLINE
 #else
-#  if defined(_MSC_VER)
-#    define KWK_FORCEINLINE __forceinline
-#  elif defined(__GNUC__) && __GNUC__ > 3
+#  if defined(__GNUC__) && (__GNUC__ > 3) || defined(__clang__) // Clang-CL does not define __GNUC__ https://github.com/llvm/llvm-project/issues/53259
```

Comment:
I am thinking that people won't compile this with gcc 2.95.2 so the > 3 is maybe useless ?

Reply:
i do too but it was already there so i left it :P

Reply:
Ok, we'll clean that up later